.rspec

```diff
@@ -0,0 +1,5 @@
+--require spec_helper
+--require rails_helper
+--require helpers/matcher_helper
```

Comment:
are you sure both lines are required?
Usually, this `.rspec` helper is just to let rspec know which files to load at start. You shouldn't "abuse" it and replace your `require`s with it.